### ISSUE TITLE
run-s is missing

### ISSUE THREAD

**Issue by CaledoniaProject** [BloodHoundAD/BloodHound#655]
`run-s` is missing on nodejs, I had to replace it with `npm run -s`, is that expected behavior?

**Comment by JonasBK** [comment]
Hi @CaledoniaProject,

No, that is not expected. I do not have that issue on Windows with node.js v16.15.0.
What version of node.js are you running and what OS?

**Comment by CaledoniaProject** [comment]
@JonasBK Running homebrew nodejs 14.X on MacOS

**Comment by JonasBK** [comment]
@CaledoniaProject hmm.. The neo4j install instructions say, 'Do not install from brew':
![image](https://user-images.githubusercontent.com/12843299/235435854-ce916b46-a883-4b7d-b29e-ba1132700ee1.png)

I wonder if it could be an issue with homebrew for nodejs as well..

I am not a macOS user, so I am just guessing here..

**Comment by CaledoniaProject** [comment]
This has nothing to do with neo4j, it's a nodejs packages.json issue

**Comment by JonasBK** [comment]
Yes, I was wondering if nodejs should not be installed using homebrew, just like the neo4j should not.
